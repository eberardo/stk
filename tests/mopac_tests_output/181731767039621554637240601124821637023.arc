

                     SUMMARY OF PM7 CALCULATION, Site No: 23169

                                                       MOPAC2016 (Version: 16.175M)
                                                       Wed May 10 12:35:58 2017
                                                       No. of days remaining =  45

           Empirical Formula: C6 H14 N2  =    22 atoms

PM7 OPT GNORM=0.01  EPS=80.1  CHARGE=0  PDBOUT
 181731767039621554637240601124821637023



     GEOMETRY OPTIMISED USING EIGENVECTOR FOLLOWING (EF).     
     SCF FIELD WAS ACHIEVED                                   

          HEAT OF FORMATION       =        -25.54415 KCAL/MOL =    -106.87671 KJ/MOL
          TOTAL ENERGY            =      -1299.39021 EV
          ELECTRONIC ENERGY       =      -2024.41445 EV
          CORE-CORE REPULSION     =        725.02424 EV
          DIELECTRIC ENERGY       =         -0.53451 EV
          GRADIENT NORM           =          5.05461
          DIPOLE                  =          3.51607 DEBYE    POINT GROUP:       C1  
          NO. OF FILLED LEVELS    =         24
          IONIZATION POTENTIAL    =          9.646896 EV
          HOMO LUMO ENERGIES (EV) =         -9.647  3.092
          MOLECULAR WEIGHT        =        114.1900
          COSMO AREA              =        158.84 SQUARE ANGSTROMS
          COSMO VOLUME            =        138.02 CUBIC ANGSTROMS

          MOLECULAR DIMENSIONS (Angstroms)

            Atom       Atom       Distance
            H     5    H    21     5.88200
            H     4    H    19     5.30624
            H    14    H     7     2.78446
          SCF CALCULATIONS        =         33
          WALL-CLOCK TIME         =          0.316 SECONDS
          COMPUTATION TIME        =          1.826 SECONDS


          FINAL GEOMETRY OBTAINED
PM7 OPT GNORM=0.01  EPS=80.1  CHARGE=0  PDBOUT
 181731767039621554637240601124821637023

  C(HETATM    1  C   HET A   1)  -0.22418412 +1  -0.67565382 +1  74.45658665 +1
  H(HETATM    2  H   HET A   1)  -1.31715163 +1  -0.53033853 +1  74.55758326 +1
  C(HETATM    3  C   HET A   1)   0.48698340 +1   0.68313232 +1  74.41818721 +1
  H(HETATM    4  H   HET A   1)   0.08598240 +1  -1.24159184 +1  75.35606311 +1
  H(HETATM    5  H   HET A   1)   0.23483932 +1   1.26164847 +1  75.32577787 +1
  C(HETATM    6  C   HET A   1)   0.09949399 +1   1.47631957 +1  73.16860692 +1
  H(HETATM    7  H   HET A   1)   1.58341246 +1   0.53346931 +1  74.44491289 +1
  H(HETATM    8  H   HET A   1)  -0.97380029 +1   1.74438741 +1  73.19880376 +1
  C(HETATM    9  C   HET A   1)   0.41317652 +1   0.67079764 +1  71.89093745 +1
  H(HETATM   10  H   HET A   1)   0.64356343 +1   2.44151184 +1  73.14238442 +1
  C(HETATM   11  C   HET A   1)  -0.34717997 +1  -0.69001147 +1  71.94668747 +1
  H(HETATM   12  H   HET A   1)   1.51206395 +1   0.47475936 +1  71.82293888 +1
  C(HETATM   13  C   HET A   1)   0.08019964 +1  -1.48756183 +1  73.19398591 +1
  H(HETATM   14  H   HET A   1)  -1.44733200 +1  -0.48383329 +1  72.00250384 +1
  H(HETATM   15  H   HET A   1)   1.15612980 +1  -1.73645150 +1  73.13803272 +1
  H(HETATM   16  H   HET A   1)  -0.44848630 +1  -2.45721074 +1  73.22511041 +1
  N(HETATM   17  N   HET A   1)   0.02071512 +1   1.50234777 +1  70.72576696 +1
  N(HETATM   18  N   HET A   1)  -0.00158369 +1  -1.48468773 +1  70.73735191 +1
  H(HETATM   19  H   HET A   1)   0.36465705 +1   1.10409223 +1  69.85725955 +1
  H(HETATM   20  H   HET A   1)  -0.98838147 +1   1.57168491 +1  70.64820665 +1
  H(HETATM   21  H   HET A   1)  -0.44119173 +1  -2.39713878 +1  70.77010198 +1
  H(HETATM   22  H   HET A   1)  -0.33154747 +1  -1.02329630 +1  69.89819423 +1
 
