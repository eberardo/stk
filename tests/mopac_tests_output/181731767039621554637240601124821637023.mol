
     RDKit          3D

  0  0  0  0  0  0  0  0  0  0999 V3000
M  V30 BEGIN CTAB
M  V30 COUNTS 22 22 0 0 0
M  V30 BEGIN ATOM
M  V30 1 C -0.2222 -0.6743 74.4672 0
M  V30 2 H -1.2978 -0.5245 74.5704 0
M  V30 3 C 0.4891 0.6869 74.4408 0
M  V30 4 H 0.0962 -1.2442 75.3409 0
M  V30 5 H 0.2363 1.2602 75.3334 0
M  V30 6 C 0.1119 1.4813 73.1818 0
M  V30 7 H 1.5695 0.5370 74.4660 0
M  V30 8 H -0.9447 1.7480 73.2242 0
M  V30 9 C 0.4059 0.6805 71.9024 0
M  V30 10 H 0.6655 2.4208 73.1643 0
M  V30 11 C -0.3198 -0.6759 71.9285 0
M  V30 12 H 1.4827 0.5029 71.8623 0
M  V30 13 C 0.0678 -1.4715 73.1863 0
M  V30 14 H -1.3964 -0.4990 71.9696 0
M  V30 15 H 1.1292 -1.7228 73.1504 0
M  V30 16 H -0.4725 -2.4181 73.2177 0
M  V30 17 N -0.0000 1.4302 70.7107 0
M  V30 18 N 0.0000 -1.4302 70.7107 0
M  V30 19 H 0.3733 0.9519 69.8925 0
M  V30 20 H -1.0136 1.4222 70.6232 0
M  V30 21 H -0.4549 -2.3417 70.7245 0
M  V30 22 H -0.3523 -0.9377 69.8935 0
M  V30 END ATOM
M  V30 BEGIN BOND
M  V30 0 1 1 2
M  V30 1 1 1 3
M  V30 2 1 1 4
M  V30 3 1 1 13
M  V30 4 1 3 5
M  V30 5 1 3 6
M  V30 6 1 3 7
M  V30 7 1 6 8
M  V30 8 1 6 9
M  V30 9 1 6 10
M  V30 10 1 9 11
M  V30 11 1 9 12
M  V30 12 1 9 17
M  V30 13 1 11 14
M  V30 14 1 11 13
M  V30 15 1 11 18
M  V30 16 1 13 15
M  V30 17 1 13 16
M  V30 18 1 17 19
M  V30 19 1 17 20
M  V30 20 1 18 21
M  V30 21 1 18 22
M  V30 END BOND
M  V30 END CTAB
M  END

$$$$
