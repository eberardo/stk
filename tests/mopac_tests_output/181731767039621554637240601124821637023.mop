PM7 OPT GNORM=0.01  EPS=80.1  CHARGE=0  PDBOUT
181731767039621554637240601124821637023

C   -0.2222   +1  -0.6743   +1  74.4672   +1 
H   -1.2978   +1  -0.5245   +1  74.5704   +1 
C   0.4891   +1  0.6869   +1  74.4408   +1 
H   0.0962   +1  -1.2442   +1  75.3409   +1 
H   0.2363   +1  1.2602   +1  75.3334   +1 
C   0.1119   +1  1.4813   +1  73.1818   +1 
H   1.5695   +1  0.537   +1  74.466   +1 
H   -0.9447   +1  1.748   +1  73.2242   +1 
C   0.4059   +1  0.6805   +1  71.9024   +1 
H   0.6655   +1  2.4208   +1  73.1643   +1 
C   -0.3198   +1  -0.6759   +1  71.9285   +1 
H   1.4827   +1  0.5029   +1  71.8623   +1 
C   0.0678   +1  -1.4715   +1  73.1863   +1 
H   -1.3964   +1  -0.499   +1  71.9696   +1 
H   1.1292   +1  -1.7228   +1  73.1504   +1 
H   -0.4725   +1  -2.4181   +1  73.2177   +1 
N   -0.0   +1  1.4302   +1  70.7107   +1 
N   0.0   +1  -1.4302   +1  70.7107   +1 
H   0.3733   +1  0.9519   +1  69.8925   +1 
H   -1.0136   +1  1.4222   +1  70.6232   +1 
H   -0.4549   +1  -2.3417   +1  70.7245   +1 
H   -0.3523   +1  -0.9377   +1  69.8935   +1 
