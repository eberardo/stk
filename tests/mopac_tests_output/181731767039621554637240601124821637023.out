 *******************************************************************************
 ** Site#: 23169        For non-commercial use only     Version 16.175M 64BITS**
 *******************************************************************************
 ** Cite this program as: MOPAC2016, Version: 16.175M, James J. P. Stewart,   **
 ** Stewart Computational Chemistry, web: HTTP://OpenMOPAC.net. Days left:  45**
 *******************************************************************************
 **                                                                           **
 **                                MOPAC2016                                  **
 **                                                                           **
 *******************************************************************************

                              PM7 CALCULATION RESULTS


 *******************************************************************************
 *  CALCULATION DONE:                                Wed May 10 12:35:58 2017  *
 *  PM7        - The PM7 Hamiltonian to be used
 *  PDBOUT     - PRINT GEOMETY IN PDB FORMAT
 *  OPT        - OPTIMIZE COORDINATES OF ALL ATOMS
 *
 *
 *
 *                 CHARGE ON SYSTEM = 0
 *
 *
 *
 *  EPS= 80.10 - USE ANDREAS KLAMT'S COSMO IMPLICIT SOLVATION MODEL
 *  T=         - A TIME OF 172800.0 SECONDS REQUESTED
 *  DUMP=N     - RESTART FILE WRITTEN EVERY  7200.000 SECONDS
 *  GNORM=     - EXIT WHEN GRADIENT NORM DROPS BELOW  0.100E-01
 *******************************************************************************
PM7 OPT GNORM=0.01  EPS=80.1  CHARGE=0  PDBOUT
 181731767039621554637240601124821637023

   ATOM   CHEMICAL          X               Y               Z
  NUMBER   SYMBOL      (ANGSTROMS)     (ANGSTROMS)     (ANGSTROMS)
 
     1       C         -0.22220000  *  -0.67430000  *  74.46720000  *
     2       H         -1.29780000  *  -0.52450000  *  74.57040000  *
     3       C          0.48910000  *   0.68690000  *  74.44080000  *
     4       H          0.09620000  *  -1.24420000  *  75.34090000  *
     5       H          0.23630000  *   1.26020000  *  75.33340000  *
     6       C          0.11190000  *   1.48130000  *  73.18180000  *
     7       H          1.56950000  *   0.53700000  *  74.46600000  *
     8       H         -0.94470000  *   1.74800000  *  73.22420000  *
     9       C          0.40590000  *   0.68050000  *  71.90240000  *
    10       H          0.66550000  *   2.42080000  *  73.16430000  *
    11       C         -0.31980000  *  -0.67590000  *  71.92850000  *
    12       H          1.48270000  *   0.50290000  *  71.86230000  *
    13       C          0.06780000  *  -1.47150000  *  73.18630000  *
    14       H         -1.39640000  *  -0.49900000  *  71.96960000  *
    15       H          1.12920000  *  -1.72280000  *  73.15040000  *
    16       H         -0.47250000  *  -2.41810000  *  73.21770000  *
    17       N          0.00000000  *   1.43020000  *  70.71070000  *
    18       N          0.00000000  *  -1.43020000  *  70.71070000  *
    19       H          0.37330000  *   0.95190000  *  69.89250000  *
    20       H         -1.01360000  *   1.42220000  *  70.62320000  *
    21       H         -0.45490000  *  -2.34170000  *  70.72450000  *
    22       H         -0.35230000  *  -0.93770000  *  69.89350000  *
 


          CARTESIAN COORDINATES 

    NO.       ATOM         X         Y         Z

     1         C       -0.2222   -0.6743   74.4672
     2         H       -1.2978   -0.5245   74.5704
     3         C        0.4891    0.6869   74.4408
     4         H        0.0962   -1.2442   75.3409
     5         H        0.2363    1.2602   75.3334
     6         C        0.1119    1.4813   73.1818
     7         H        1.5695    0.5370   74.4660
     8         H       -0.9447    1.7480   73.2242
     9         C        0.4059    0.6805   71.9024
    10         H        0.6655    2.4208   73.1643
    11         C       -0.3198   -0.6759   71.9285
    12         H        1.4827    0.5029   71.8623
    13         C        0.0678   -1.4715   73.1863
    14         H       -1.3964   -0.4990   71.9696
    15         H        1.1292   -1.7228   73.1504
    16         H       -0.4725   -2.4181   73.2177
    17         N        0.0000    1.4302   70.7107
    18         N        0.0000   -1.4302   70.7107
    19         H        0.3733    0.9519   69.8925
    20         H       -1.0136    1.4222   70.6232
    21         H       -0.4549   -2.3417   70.7245
    22         H       -0.3523   -0.9377   69.8935

 General Reference for PM7:
 "Optimization of Parameters for Semiempirical Methods VI: More Modifications to the 
 NDDO Approximations and Re-optimization of Parameters", J. J. P. Stewart, J. Mol. Mod., 1:32, 19 (2013)
 http://www.springerlink.com/openurl.asp?genre=article&id=doi:10.1007/s00894-012-1667-x

           Empirical Formula: C6 H14 N2  =    22 atoms



      MOLECULAR POINT GROUP   :   C1  


      RHF CALCULATION, NO. OF DOUBLY OCCUPIED LEVELS = 24


          Geometry optimization using EF

          DIAGONAL MATRIX USED AS START HESSIAN

 CYCLE:     1 TIME:   0.016 TIME LEFT:  2.00D  GRAD.:    61.292 HEAT: -21.68341
 CYCLE:     2 TIME:   0.027 TIME LEFT:  2.00D  GRAD.:    36.379 HEAT: -23.18263
 CYCLE:     3 TIME:   0.012 TIME LEFT:  2.00D  GRAD.:    32.504 HEAT: -24.01555
 CYCLE:     4 TIME:   0.020 TIME LEFT:  2.00D  GRAD.:    23.684 HEAT: -24.22538
 CYCLE:     5 TIME:   0.008 TIME LEFT:  2.00D  GRAD.:    25.743 HEAT: -24.31253
 CYCLE:     6 TIME:   0.012 TIME LEFT:  2.00D  GRAD.:    24.808 HEAT: -24.43491
 CYCLE:     7 TIME:   0.012 TIME LEFT:  2.00D  GRAD.:    25.072 HEAT: -24.58845
 CYCLE:     8 TIME:   0.012 TIME LEFT:  2.00D  GRAD.:    12.409 HEAT: -24.93170
 CYCLE:     9 TIME:   0.020 TIME LEFT:  2.00D  GRAD.:    11.929 HEAT: -24.97342
 CYCLE:    10 TIME:   0.008 TIME LEFT:  2.00D  GRAD.:    11.486 HEAT: -25.05612
 CYCLE:    11 TIME:   0.012 TIME LEFT:  2.00D  GRAD.:     8.922 HEAT: -25.13097
 CYCLE:    12 TIME:   0.008 TIME LEFT:  2.00D  GRAD.:    13.961 HEAT: -25.23764
 CYCLE:    13 TIME:   0.012 TIME LEFT:  2.00D  GRAD.:    19.626 HEAT: -25.38825
 CYCLE:    14 TIME:   0.008 TIME LEFT:  2.00D  GRAD.:    14.026 HEAT: -25.41080
 CYCLE:    15 TIME:   0.020 TIME LEFT:  2.00D  GRAD.:     9.876 HEAT: -25.51628
 CYCLE:    16 TIME:   0.008 TIME LEFT:  2.00D  GRAD.:     7.603 HEAT: -25.52792
 CYCLE:    17 TIME:   0.039 TIME LEFT:  2.00D  GRAD.:     5.906 HEAT: -25.54001
 CYCLE:    18 TIME:   0.027 TIME LEFT:  2.00D  GRAD.:     5.055 HEAT: -25.54415

     TRUST RADIUS NOW LESS THAN  0.00010 OPTIMIZATION TERMINATING
      THE GEOMETRY MAY NOT BE COMPLETELY OPTIMIZED
      (TO CONTINUE, ADD 'LET DDMIN=0.0' TO THE KEYWORD LINE)

 -------------------------------------------------------------------------------
PM7 OPT GNORM=0.01  EPS=80.1  CHARGE=0  PDBOUT
 181731767039621554637240601124821637023



     GEOMETRY OPTIMISED USING EIGENVECTOR FOLLOWING (EF).     
     SCF FIELD WAS ACHIEVED                                   


                              PM7 CALCULATION
                                                       MOPAC2016 (Version: 16.175M)
                                                       Wed May 10 12:35:58 2017
                                                       No. of days remaining =  45




          FINAL HEAT OF FORMATION =        -25.54415 KCAL/MOL =    -106.87671 KJ/MOL
          VAN DER WAALS AREA      =        158.84 SQUARE ANGSTROMS


          TOTAL ENERGY            =      -1299.39021 EV
          ELECTRONIC ENERGY       =      -2024.41445 EV  POINT GROUP:     C1  
          CORE-CORE REPULSION     =        725.02424 EV
          DIELECTRIC ENERGY       =         -0.53451 EV
          COSMO AREA              =        158.84 SQUARE ANGSTROMS
          COSMO VOLUME            =        138.02 CUBIC ANGSTROMS

          GRADIENT NORM           =          5.05461
          IONIZATION POTENTIAL    =          9.646896 EV
          HOMO LUMO ENERGIES (EV) =         -9.647  3.092
          NO. OF FILLED LEVELS    =         24
          MOLECULAR WEIGHT        =        114.1900

          MOLECULAR DIMENSIONS (Angstroms)

            Atom       Atom       Distance
            H     5    H    21     5.88200
            H     4    H    19     5.30624
            H    14    H     7     2.78446


          SCF CALCULATIONS        =         33
 
          WALL-CLOCK TIME         =          0.316 SECONDS
          COMPUTATION TIME        =          1.821 SECONDS



       FINAL  POINT  AND  DERIVATIVES

   PARAMETER     ATOM    TYPE            VALUE       GRADIENT
      1          1  C    CARTESIAN X    -0.224184     0.187737  KCAL/ANGSTROM
      2          1  C    CARTESIAN Y    -0.675654     0.280762  KCAL/ANGSTROM
      3          1  C    CARTESIAN Z    74.456587     0.073337  KCAL/ANGSTROM
      4          2  H    CARTESIAN X    -1.317152    -0.045198  KCAL/ANGSTROM
      5          2  H    CARTESIAN Y    -0.530339    -0.359212  KCAL/ANGSTROM
      6          2  H    CARTESIAN Z    74.557583     0.005707  KCAL/ANGSTROM
      7          3  C    CARTESIAN X     0.486983    -0.675881  KCAL/ANGSTROM
      8          3  C    CARTESIAN Y     0.683132     1.636803  KCAL/ANGSTROM
      9          3  C    CARTESIAN Z    74.418187     0.315144  KCAL/ANGSTROM
     10          4  H    CARTESIAN X     0.085982    -0.043543  KCAL/ANGSTROM
     11          4  H    CARTESIAN Y    -1.241592    -0.157611  KCAL/ANGSTROM
     12          4  H    CARTESIAN Z    75.356063     0.334485  KCAL/ANGSTROM
     13          5  H    CARTESIAN X     0.234839    -0.325067  KCAL/ANGSTROM
     14          5  H    CARTESIAN Y     1.261648    -0.724901  KCAL/ANGSTROM
     15          5  H    CARTESIAN Z    75.325778    -0.613893  KCAL/ANGSTROM
     16          6  C    CARTESIAN X     0.099494     0.084678  KCAL/ANGSTROM
     17          6  C    CARTESIAN Y     1.476320    -1.012696  KCAL/ANGSTROM
     18          6  C    CARTESIAN Z    73.168607     0.457542  KCAL/ANGSTROM
     19          7  H    CARTESIAN X     1.583412    -0.110436  KCAL/ANGSTROM
     20          7  H    CARTESIAN Y     0.533469     0.006693  KCAL/ANGSTROM
     21          7  H    CARTESIAN Z    74.444913    -0.209621  KCAL/ANGSTROM
     22          8  H    CARTESIAN X    -0.973800     0.051066  KCAL/ANGSTROM
     23          8  H    CARTESIAN Y     1.744387     0.702832  KCAL/ANGSTROM
     24          8  H    CARTESIAN Z    73.198804    -0.023746  KCAL/ANGSTROM
     25          9  C    CARTESIAN X     0.413177     0.424171  KCAL/ANGSTROM
     26          9  C    CARTESIAN Y     0.670798    -0.946473  KCAL/ANGSTROM
     27          9  C    CARTESIAN Z    71.890937    -0.605158  KCAL/ANGSTROM
     28         10  H    CARTESIAN X     0.643563     0.333122  KCAL/ANGSTROM
     29         10  H    CARTESIAN Y     2.441512     0.939086  KCAL/ANGSTROM
     30         10  H    CARTESIAN Z    73.142384     0.104765  KCAL/ANGSTROM
     31         11  C    CARTESIAN X    -0.347180     0.016277  KCAL/ANGSTROM
     32         11  C    CARTESIAN Y    -0.690011     0.352868  KCAL/ANGSTROM
     33         11  C    CARTESIAN Z    71.946687    -0.151293  KCAL/ANGSTROM
     34         12  H    CARTESIAN X     1.512064     0.141674  KCAL/ANGSTROM
     35         12  H    CARTESIAN Y     0.474759     0.092366  KCAL/ANGSTROM
     36         12  H    CARTESIAN Z    71.822939     0.134113  KCAL/ANGSTROM
     37         13  C    CARTESIAN X     0.080200    -0.248860  KCAL/ANGSTROM
     38         13  C    CARTESIAN Y    -1.487562    -2.326222  KCAL/ANGSTROM
     39         13  C    CARTESIAN Z    73.193986     0.234877  KCAL/ANGSTROM
     40         14  H    CARTESIAN X    -1.447332    -0.159210  KCAL/ANGSTROM
     41         14  H    CARTESIAN Y    -0.483833     0.486909  KCAL/ANGSTROM
     42         14  H    CARTESIAN Z    72.002504     0.425869  KCAL/ANGSTROM
     43         15  H    CARTESIAN X     1.156130    -0.278772  KCAL/ANGSTROM
     44         15  H    CARTESIAN Y    -1.736451    -0.677256  KCAL/ANGSTROM
     45         15  H    CARTESIAN Z    73.138033    -0.101935  KCAL/ANGSTROM
     46         16  H    CARTESIAN X    -0.448486     0.504063  KCAL/ANGSTROM
     47         16  H    CARTESIAN Y    -2.457211     0.738812  KCAL/ANGSTROM
     48         16  H    CARTESIAN Z    73.225110    -0.060059  KCAL/ANGSTROM
     49         17  N    CARTESIAN X     0.020715    -1.368164  KCAL/ANGSTROM
     50         17  N    CARTESIAN Y     1.502348     1.207485  KCAL/ANGSTROM
     51         17  N    CARTESIAN Z    70.725767     0.133736  KCAL/ANGSTROM
     52         18  N    CARTESIAN X    -0.001584    -0.899496  KCAL/ANGSTROM
     53         18  N    CARTESIAN Y    -1.484688    -0.640651  KCAL/ANGSTROM
     54         18  N    CARTESIAN Z    70.737352    -0.560716  KCAL/ANGSTROM
     55         19  H    CARTESIAN X     0.364657     0.709056  KCAL/ANGSTROM
     56         19  H    CARTESIAN Y     1.104092     0.037302  KCAL/ANGSTROM
     57         19  H    CARTESIAN Z    69.857260    -0.681641  KCAL/ANGSTROM
     58         20  H    CARTESIAN X    -0.988381     0.084758  KCAL/ANGSTROM
     59         20  H    CARTESIAN Y     1.571685    -0.832597  KCAL/ANGSTROM
     60         20  H    CARTESIAN Z    70.648207     0.014882  KCAL/ANGSTROM
     61         21  H    CARTESIAN X    -0.441192     0.804951  KCAL/ANGSTROM
     62         21  H    CARTESIAN Y    -2.397139     0.832323  KCAL/ANGSTROM
     63         21  H    CARTESIAN Z    70.770102     0.091371  KCAL/ANGSTROM
     64         22  H    CARTESIAN X    -0.331547     0.813070  KCAL/ANGSTROM
     65         22  H    CARTESIAN Y    -1.023296     0.363348  KCAL/ANGSTROM
     66         22  H    CARTESIAN Z    69.898194     0.682204  KCAL/ANGSTROM




   ATOM                 CHEMICAL                   X               Y               Z
  NUMBER                 SYMBOL               (ANGSTROMS)     (ANGSTROMS)     (ANGSTROMS)
 
     1       C(HETATM    1  C   HET A   1)   -0.22418412  *  -0.67565382  *  74.45658665  *
     2       H(HETATM    2  H   HET A   1)   -1.31715163  *  -0.53033853  *  74.55758326  *
     3       C(HETATM    3  C   HET A   1)    0.48698340  *   0.68313232  *  74.41818721  *
     4       H(HETATM    4  H   HET A   1)    0.08598240  *  -1.24159184  *  75.35606311  *
     5       H(HETATM    5  H   HET A   1)    0.23483932  *   1.26164847  *  75.32577787  *
     6       C(HETATM    6  C   HET A   1)    0.09949399  *   1.47631957  *  73.16860692  *
     7       H(HETATM    7  H   HET A   1)    1.58341246  *   0.53346931  *  74.44491289  *
     8       H(HETATM    8  H   HET A   1)   -0.97380029  *   1.74438741  *  73.19880376  *
     9       C(HETATM    9  C   HET A   1)    0.41317652  *   0.67079764  *  71.89093745  *
    10       H(HETATM   10  H   HET A   1)    0.64356343  *   2.44151184  *  73.14238442  *
    11       C(HETATM   11  C   HET A   1)   -0.34717997  *  -0.69001147  *  71.94668747  *
    12       H(HETATM   12  H   HET A   1)    1.51206395  *   0.47475936  *  71.82293888  *
    13       C(HETATM   13  C   HET A   1)    0.08019964  *  -1.48756183  *  73.19398591  *
    14       H(HETATM   14  H   HET A   1)   -1.44733200  *  -0.48383329  *  72.00250384  *
    15       H(HETATM   15  H   HET A   1)    1.15612980  *  -1.73645150  *  73.13803272  *
    16       H(HETATM   16  H   HET A   1)   -0.44848630  *  -2.45721074  *  73.22511041  *
    17       N(HETATM   17  N   HET A   1)    0.02071512  *   1.50234777  *  70.72576696  *
    18       N(HETATM   18  N   HET A   1)   -0.00158369  *  -1.48468773  *  70.73735191  *
    19       H(HETATM   19  H   HET A   1)    0.36465705  *   1.10409223  *  69.85725955  *
    20       H(HETATM   20  H   HET A   1)   -0.98838147  *   1.57168491  *  70.64820665  *
    21       H(HETATM   21  H   HET A   1)   -0.44119173  *  -2.39713878  *  70.77010198  *
    22       H(HETATM   22  H   HET A   1)   -0.33154747  *  -1.02329630  *  69.89819423  *

                             CARTESIAN COORDINATES

   1    C       -0.224184121    -0.675653819    74.456586650
   2    H       -1.317151632    -0.530338529    74.557583261
   3    C        0.486983400     0.683132320    74.418187213
   4    H        0.085982401    -1.241591836    75.356063106
   5    H        0.234839323     1.261648471    75.325777870
   6    C        0.099493987     1.476319568    73.168606925
   7    H        1.583412459     0.533469312    74.444912889
   8    H       -0.973800294     1.744387411    73.198803757
   9    C        0.413176519     0.670797640    71.890937449
  10    H        0.643563434     2.441511840    73.142384421
  11    C       -0.347179971    -0.690011474    71.946687474
  12    H        1.512063954     0.474759360    71.822938876
  13    C        0.080199638    -1.487561825    73.193985909
  14    H       -1.447332003    -0.483833293    72.002503842
  15    H        1.156129805    -1.736451496    73.138032720
  16    H       -0.448486302    -2.457210743    73.225110405
  17    N        0.020715119     1.502347767    70.725766958
  18    N       -0.001583688    -1.484687732    70.737351905
  19    H        0.364657048     1.104092228    69.857259554
  20    H       -0.988381472     1.571684914    70.648206646
  21    H       -0.441191732    -2.397138780    70.770101983
  22    H       -0.331547468    -1.023296297    69.898194227


           Empirical Formula: C6 H14 N2  =    22 atoms



      MOLECULAR POINT GROUP   :   C1  


                  EIGENVALUES  
 -36.17631 -34.74328 -29.64916 -25.59564 -24.72927 -20.81651 -20.38955 -19.57063
 -16.76203 -16.65386 -16.31327 -15.76265 -15.06392 -14.45664 -13.90649 -13.25724
 -13.06603 -12.46179 -12.07594 -11.54169 -11.36012 -11.25778 -10.04369  -9.64690
   3.09171   3.45479   3.84943   4.10671   4.17300   4.22419   4.42714   4.80341
   5.59731   5.61748   5.66251   5.85229   5.95123   5.97477   6.10726   6.28384
   6.51593   6.62139   6.64368   6.67344   6.95308   6.99552


              NET ATOMIC CHARGES AND DIPOLE CONTRIBUTIONS

  ATOM NO.                 TYPE                        CHARGE      No. of ELECS.   s-Pop       p-Pop       
    1          C(HETATM    1  C   HET A   1)          -0.272686        4.2727     1.05872     3.21397
    2          H(HETATM    2  H   HET A   1)           0.143088        0.8569     0.85691
    3          C(HETATM    3  C   HET A   1)          -0.270481        4.2705     1.05805     3.21243
    4          H(HETATM    4  H   HET A   1)           0.141476        0.8585     0.85852
    5          H(HETATM    5  H   HET A   1)           0.140421        0.8596     0.85958
    6          C(HETATM    6  C   HET A   1)          -0.295850        4.2959     1.06399     3.23187
    7          H(HETATM    7  H   HET A   1)           0.141346        0.8587     0.85865
    8          H(HETATM    8  H   HET A   1)           0.148057        0.8519     0.85194
    9          C(HETATM    9  C   HET A   1)           0.041124        3.9589     1.07830     2.88058
   10          H(HETATM   10  H   HET A   1)           0.146782        0.8532     0.85322
   11          C(HETATM   11  C   HET A   1)           0.001607        3.9984     1.08184     2.91656
   12          H(HETATM   12  H   HET A   1)           0.148052        0.8519     0.85195
   13          C(HETATM   13  C   HET A   1)          -0.292030        4.2920     1.06324     3.22879
   14          H(HETATM   14  H   HET A   1)           0.128335        0.8717     0.87166
   15          H(HETATM   15  H   HET A   1)           0.150650        0.8494     0.84935
   16          H(HETATM   16  H   HET A   1)           0.145033        0.8550     0.85497
   17          N(HETATM   17  N   HET A   1)          -0.708698        5.7087     1.65713     4.05157
   18          N(HETATM   18  N   HET A   1)          -0.711116        5.7111     1.65778     4.05333
   19          H(HETATM   19  H   HET A   1)           0.267535        0.7325     0.73247
   20          H(HETATM   20  H   HET A   1)           0.270673        0.7293     0.72933
   21          H(HETATM   21  H   HET A   1)           0.271618        0.7284     0.72838
   22          H(HETATM   22  H   HET A   1)           0.265067        0.7349     0.73493

          NET CHARGE ON RESIDUES

      Residue         Charge  Anion or
                              Cation?
      HET A   1       +0.000
 


 DIPOLE           X         Y         Z       TOTAL
 POINT-CHG.    -1.538    -0.860     1.294     2.186
 HYBRID        -1.488    -0.865    -0.815     1.905
 SUM           -3.026    -1.725     0.479     3.516


          ATOMIC ORBITAL ELECTRON POPULATIONS

                   Atom                  s        px        py        pz   
   C(HETATM    1  C   HET A   1)   1.05872   1.13095   1.02528   1.05774
   H(HETATM    2  H   HET A   1)   0.85691
   C(HETATM    3  C   HET A   1)   1.05805   1.12397   1.02727   1.06119
   H(HETATM    4  H   HET A   1)   0.85852
   H(HETATM    5  H   HET A   1)   0.85958
   C(HETATM    6  C   HET A   1)   1.06399   1.16132   1.07876   0.99178
   H(HETATM    7  H   HET A   1)   0.85865
   H(HETATM    8  H   HET A   1)   0.85194
   C(HETATM    9  C   HET A   1)   1.07830   1.08025   0.91235   0.88798
   H(HETATM   10  H   HET A   1)   0.85322
   C(HETATM   11  C   HET A   1)   1.08184   1.07319   0.95761   0.88576
   H(HETATM   12  H   HET A   1)   0.85195
   C(HETATM   13  C   HET A   1)   1.06324   1.16111   1.07612   0.99156
   H(HETATM   14  H   HET A   1)   0.87166
   H(HETATM   15  H   HET A   1)   0.84935
   H(HETATM   16  H   HET A   1)   0.85497
   N(HETATM   17  N   HET A   1)   1.65713   1.29901   1.57545   1.17711
   N(HETATM   18  N   HET A   1)   1.65778   1.68768   1.20452   1.16113
   H(HETATM   19  H   HET A   1)   0.73247
   H(HETATM   20  H   HET A   1)   0.72933
   H(HETATM   21  H   HET A   1)   0.72838
   H(HETATM   22  H   HET A   1)   0.73493

 **********************
 *                    *
 * JOB ENDED NORMALLY *
 *                    *
 **********************



 TOTAL JOB TIME:             0.32 SECONDS

 == MOPAC DONE ==
